package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/checkout-components",
-  "version": "5.0.86",
+  "version": "5.0.87",
```

Comment:
Please back this change out. This will happen automatically at publish time. Thanks!

Reply:
Version bump rolled back.